packages/core/src/modules/dcql/DcqlService.ts

```diff
@@ -123,9 +150,17 @@ export class DcqlService {
         if (result.success) {
           if (result.output.credential_format === 'vc+sd-jwt') {
             const sdJwtVcRecord = credentialRecords[result.input_credential_index] as SdJwtVcRecord
-            agentContext.dependencyManager
+            const claims = agentContext.dependencyManager
               .resolve(SdJwtVcService)
               .applyDisclosuresForPayload(sdJwtVcRecord.compactSdJwtVc, result.output.claims)
+            return [
+              credential_query_id,
+              {
+                ...result,
+                output: { ...result.output, claims },
+                record: credentialRecords[result.input_credential_index],
+              },
+            ]
```

Comment:
This returns the whole sd jwt vc. not the claims. 


So should do `claims.prettyClaims`